packages/components/src/tooltip/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo, useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { usePopper } from 'react-popper';
+
+import { arrayToHash } from 'nebenan-helpers/lib/data';
+import { getPopperOptions } from './utils';
+import {
+  POSITION_TOP,
+  POSITION_BOTTOM,
+  POSITION_LEFT,
+  POSITION_RIGHT,
+} from '../feature_alert';
+import styles from './index.module.scss';
+
+const KNOWN_TYPES = arrayToHash([POSITION_LEFT, POSITION_RIGHT, POSITION_TOP, POSITION_BOTTOM]);
+
+const Tooltip = (props) => {
+  const [isOpen, setOpen] = useState(false);
+  const { type, text, children, ...cleanProps } = props;
+  const selectedType = KNOWN_TYPES[type] ? type : POSITION_TOP;
```

Comment:
Would rather set POSITION_TOP as defaultProp and expect users to pass a working type